src/styles/InputList.scss

```diff
@@ -0,0 +1,12 @@
+// Style for main div
+.container {        
+    background: #fdfaf6;   
+    padding: 15%;
+    margin: auto;
+    font-family: 'EB Garamond', serif;    
+}
+// Style for input
+.form-control {
+    width: 60%;
+    margin-left: 10%;
```

Comment:
Remove all the comments, it's obvious that style is for a container and for a input :)